spec/controllers/account/companies_controller_spec.rb

```diff
@@ -0,0 +1,151 @@
+require 'rails_helper'
+
+RSpec.describe Account::CompaniesController, type: :controller do
+  let!(:user) { create(:user) }
+  before(:each) { sign_in user }
+
+  describe "GET #index" do
+    let!(:company) { create(:company) }
+
+    it "populates an array of companies" do
+      get :index
+      expect(assigns(:companies)).to eq([company])
```

Comment:
never do like this
`ActiveRecord:Association_Collection` is not an array
prefer to use `RSpec::Matchers#contain_exactly`